src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="first_form"
+      action="url"
```

Comment:
```suggestion
      action="#"
```

Reply:
action - is a link where your form will send results, and "url" is not looks like real link 🙂